Dockerfile

```diff
@@ -40,7 +40,7 @@ WORKDIR /home/epoch/node
 # Erl handle SIGQUIT instead of the default SIGINT
 STOPSIGNAL SIGQUIT
 
-EXPOSE 3013 3015 3113 3114
+EXPOSE 3013 3014 3015 3113 3114
```

Comment:
@dincho Does this require update to environments (I am thinking of something like [this](https://github.com/aeternity/infrastructure/blob/95f8b182629f265707591abd479ff3131ee2e68c/openstack/ae-environment.yml#L75))?

Reply:
@lucafavatella yes, in case we'd expose it to public in some of the environments.

Do we?